lib/web/imageRouter/index.js

```diff
@@ -49,7 +49,7 @@ imageRouter.post('/uploadimage', function (req, res) {
       rimraf(tmpDir)
       return errors.errorForbidden(res)
     } else if (!files.image || !files.image.path) {
-      logger.error(`Image upload error: Upload didn't contain file)`)
+      logger.error('Image upload error: Upload didn\'t contain file)')
```

Comment:
This bracket seems false.
```suggestion
      logger.error('Image upload error: Upload didn\'t contain file')
```

Reply:
fixed